comictaggerlib/taggerwindow.py

```diff
@@ -1538,9 +1585,13 @@ def copy_tags(self) -> None:
         has_src_count = 0
 
         src_style = self.load_data_style
-        dest_style = self.save_data_style
+        dest_styles = list(self.save_data_styles)
```

Comment:
Is there a reason for the list call here?

Reply:
No...